scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -272,17 +272,27 @@
           <input type="hidden" name="build" value="{{case.genome_build}}">
 
           <input type="hidden" name="contig" value="{% if zoom in ["start", "length"] %}{{ variant.chromosome }}{% elif zoom == "end" %}{{ variant.end_chrom }}{% endif %}">
+          {% set initial_view = namespace(start=1, end=1) %}
+
+          {% set initial_view.start = variant.position - 500 %}
+          {% set initial_view.end = variant.end + 500 %}
+
           {% if zoom == "start" %}
-            <input type="hidden" name="start" value="{{variant.position - 500}}">
-            <input type="hidden" name="stop" value="{{variant.position + 500}}">
-            <!--<input type="hidden" name="now" values="{{now}}"-->
+            {% set initial_view.end = variant.position + 500 %}
           {% elif zoom == "end" %}
-            <input type="hidden" name="start" value="{{variant.end - 500}}">
-            <input type="hidden" name="stop" value="{{variant.end + 500}}">
-          {% elif zoom == "length" %}
-            <input type="hidden" name="start" value="{{variant.position - 500}}">
-            <input type="hidden" name="stop" value="{{variant.end + 500}}">
+            {% set initial_view.start = variant.end - 500 %}
+          {% endif %}
+
+          {% if initial_view.start<1 %}
+            {% set initial_view.start = 1 %}
           {% endif %}
+
+          {% if initial_view.end<1 %}
+            {% set initial_view.end = 1 %}
```

Comment:
👍

Reply:
😊

Reply:
Better, just off by one - though it is a lot easier to see and correct now!

Reply:
It is!